app/routes/goal_routes.py

```diff
@@ -1 +1,89 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, Response, request, abort, make_response
+from ..models.goal import Goal
+from ..db import db
+from ..models.task import Task
+from .route_utilities import validate_model
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    new_goal = Goal.obj_from_dict(request_body)
+
+    db.session.add(new_goal)
+    db.session.commit()
+    response = {"goal": new_goal.obj_to_dict()}
+    return response, 201
+
+@bp.post("/<goal_id>/tasks")
+def add_tasks_to_goals(goal_id):
+    goal = validate_model(Goal, goal_id)
+    request_body = request.get_json()
+    task_ids = request_body["task_ids"]
+    tasks = Task.query.filter(Task.id.in_(task_ids)).all()
```

Comment:
The `<Model>.query` syntax has been deprecated since SQLAlchemy 2.0 and should not be used for new development. 

Review the docs [here](https://docs.sqlalchemy.org/en/20/core/selectable.html#sqlalchemy.sql.expression.Select.filter)

Here, I'd probably use your validate_model method since `task_ids` are values that are grabbed from the request body that the client sent with the request. Since we can't assume that they're valid, we can validate:
```suggestion
    tasks = [validate_model(Task, task_id) for task_id in task_ids]
```